build.fsx

```diff
@@ -494,24 +494,24 @@ Target "PublishMntr" (fun _ ->
     if not skipBuild.Value then
         let executableProjects = !! "./src/**/Akka.MultiNodeTestRunner.csproj"
```

Comment:
I need to add a switch that does a Linux version of this, on-demand, and then update our build pipelines for it